src/utils/azure_storage.ts

```diff
@@ -186,6 +187,35 @@ export const getBlobAsText = (
     );
   });
 
+/**
+ * Get a blob content as text (string). In case of error, it will return error details.
+ *
+ * @param blobService     the Azure blob service
+ * @param containerName   the name of the Azure blob storage container
+ * @param blobName        blob file name
+ */
+export const getBlobAsTextWithError = (
```

Comment:
How did you test it? Maybe it's better to add an integration test

Reply:
added for whole getContentBlob